projects/libavc/avc_dec_fuzzer.dict

```diff
@@ -0,0 +1,2 @@
+# Start code (bytes 0-3)
```

Comment:
In future, please move to <source code repo>/libavc/fuzzer/avc_dec_fuzzer.dict.
Checkout https://google.github.io/oss-fuzz/advanced-topics/ideal-integration

Reply:
Will do this and submit a PR once avc_dec_fuzzer.dict is merged in libavc repo.